src/include/duckdb/storage/serialization/nodes.json

```diff
@@ -705,7 +705,7 @@
       },
       {"id": 114,
         "name": "buffer_size",
-        "type": "idx_t"
+        "type": "CSVOption<idx_t>"
```

Comment:
Changing the type here will break serialization

Reply:
Uf, my bad,  I totally forgot about this change